app/validators.py

```diff
@@ -0,0 +1,16 @@
+def is_valid_skill(skill):
+    valid_skills = ["BJJ", "Karate", "Judo", "KungFu", "Capoeira", "Boxing", "Taekwondo", "Aikido", "KravMaga",
+                    "MuayThai", "KickBoxing", "Pankration", "Wrestling", "Sambo", "Savate", "Sumo", "Kendo",
+                    "Hapkido", "LutaLivre", "WingChu", "Ninjutsu", "Fencing", "ArmWrestling", "SuckerPunch",
+                    "44Magnum"]
+    return skill in valid_skills
```

Comment:
This can be done in O(1) time complexity using a hash map.

Reply:
I am happy to know it. Sure, I will implement it with O(1) time complexity.